### PR TITLE
Rahti1 templates

### PR DESCRIPTION
**Pull request by trispera**
## Proposed changes

Publish a CSCfi repo with some of the Rahti 1 templates.
Add a link in the documentation

https://csc-guide-preview.rahtiapp.fi/origin/rahti1-templates/cloud/rahti2/catalog-docs/

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/rahti2/catalog-docs.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ You can browse the catalog from the webinterface after (1) logging in Rahti 2 an
 ![+Add](../img/rahti-catalog.png)
 
 
-!!! warning "Not all Rahti 1 templates have been ported to Rahti 2"
-
-    The process of adapting and testing the templates to Rahti 2 is labor intensive and not in the top of our task list. If you have any special interest on the functionality provided by a Rahti 1 template that is missing in Rahti 2 please let us know (<servicedesk@csc.fi>).
+!!! Note "Rahti 1 templates in Rahti 2"
+    Most of the previous templates have been migrating to Helm Charts for Rahti 2. If you want to use a template from Rahti 1 on Rahti 2, you can find the repo at this [link](https://github.com/CSCfi/rahti-1-templates).
```
Comment (lvarin) [inline docs/cloud/rahti2/catalog-docs.md:17]:
```suggestion
    Most of the previous templates have been migrated to Helm Charts for Rahti 2. If you want to use a Rahti 1 template on Rahti 2, you can find the repo at this [link](https://github.com/CSCfi/rahti-1-templates).  
```

**Review by lvarin — CHANGES_REQUESTED** [review]
small reword

**Comment by trispera** [comment]
Suggestions committed

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**